web/blog/2023-03-03-ten-hard-truths-junior-developers-need-to-hear.md

```diff
@@ -0,0 +1,51 @@
+
+So, I put together this list of things you should consider when starting out a career in tech: 
+
+1. 👎 If you’re doing it solely for the money, you’re not gonna make it. True, you don’t need a degree or anyone’s permission to advance in this career, but you need ambition and mental stamina. A genuine interest is needed to maintain them. 
+
+2. 😎 You don’t have to follow the trends. Follow what interests you. Like I said before, you need mental stamina in this field of work. Following your interests will keep you engaged and help avoid burnout.
+
+3. 👩‍💻 You don’t need to know a piece of tech inside and out, contrary to what some devs might want you to believe. The truth is, you are always learning, and there will always be gaps in your knowledge. Your confidence in being able to fill those gaps is what matters.
+
+4. 🧱 Start building, ASAP. Find a problem that interests you and build the solution yourself. Contribute to Open-Source projects that you use. A portfolio of unique work speaks volumes about your abilities. Plus, there’s no better teacher than experience. 
+
+5. 😱 Be fearless and seek feedback. Put your work out there and be ready to have it criticized. If you can stomach it, you’ll come out the other side a much better developer. 
+
+6. 🧐 You should have a firm understanding of what you’re doing. Don’t copy-paste someone else’s answer (or GPT’s) to your problem and call it a day. Question why things work, and figure it out for yourself.
+
+7. 🏋️‍♀️ You have to do the grunt work, unfortunately. Don’t expect high salaries from the beginning. And you’ll probably want to improve your portfolio by working on side projects in your free time, or you might stay a junior dev for longer than you wish. 
+
+8. 🧗‍♂️ Challenge yourself. Don’t let yourself get too comfortable. If you do, you won’t improve. Offer to take new, difficult, and daunting tasks at work or with your personal projects. You’ll be surprised what you can achieve. 
+
+9. 💰 You don’t have to pay for boot camps or courses. In fact, you’re better off tackling problems on your own and only asking for help if you’re truly stuck. There’s a wealth of free resources out there, and when you’re on the job, these might be the only things to assist you.
+
+10. 🗣 Programming is definitely not the only skill you’ll need. Being respectful, communicative, conscientious, ambitious, and humble will put you in a different league and make you a valuable asset in any tech team. 
+
+TIP: *Contributing to an active, friendly Open-Source community project, like [https://wasp-lang.dev](https://wasp-lang.dev), is a great way to learn from others, build your skills and portfolio, and expose yourself to constructive criticism. Join us on [discord]([https://discord.gg/CFnqtaEU](https://discord.gg/CFnqtaEU))!*
```

Comment:
since this is aimed at juniors, I wonder if we should alter CTA a bit. Contributing to Wasp might be too much for them due to its complexity. On the other hand, they can contribute to docs, examples etc, that's def a good match.

Maybe the benefit could be just "joining the community of active web developers of all levels that share their experiences and build side project can be a huge boost for your motivation and improving your skillset. Join us at ..."

just brainstorming here, yours can also work, so do what you prefer.